appwidget-configuration/src/main/java/com/google/android/glance/appwidget/configuration/AppWidgetConfigurationScaffold.kt

```diff
@@ -75,7 +75,8 @@ class AppWidgetConfigurationState(
     val glanceId: GlanceId?,
     val providerInfo: AppWidgetProviderInfo?,
     val instance: GlanceAppWidget,
-    private val activity: Activity
```

Comment:
You still need the activity right? How is the code compiling without this?

Reply:
That's what I get for trying to edit this in GitHub. 🫠 I'll get this updated today.